src/node_sqlite.cc

```diff
@@ -114,6 +116,19 @@ inline void THROW_ERR_SQLITE_ERROR(Isolate* isolate, const char* message) {
   }
 }
 
+inline void THROW_ERR_SQLITE_ERROR(Isolate* isolate, int errcode) {
+  const char* errstr = sqlite3_errstr(errcode);
+  Local<Object> e;
+
+  if (CreateSQLiteError(isolate, errstr).ToLocal(&e)) {
+    e->Set(isolate->GetCurrentContext(),
+           OneByteString(isolate, "errcode"),
+           Integer::New(isolate, errcode))
+        .IsNothing();
```

Comment:
Note that this is a bit dangerous. The call to `e->Set(...)` and the call to `OneByteString(...)` could both throw their own exceptions here that really should not be ignored. The `OneByteString` can be replaced with an `env-property` string (e.g. `env()->errorcode_string()` and the `IsNothing()` should probably be a `ToChecked()` variant or something.

Reply:
Thanks!

Reply:
Addressed in https://github.com/nodejs/node/pull/56352/commits/c674327ade01df4c548f0b2c33b12c992739f991